src/components/Table/common/apy-cell.tsx

```diff
@@ -0,0 +1,110 @@
+import Decimal from "decimal.js";
+import { Box, Typography, Stack, useTheme } from "@mui/material";
+import { BsFillLightningFill } from "@react-icons/all-files/bs/BsFillLightningFill";
+
+import { useConfig } from "../../../hooks";
+import HtmlTooltip from "../../common/html-tooltip";
+import TokenIcon from "../../TokenIcon";
+import { APY_FORMAT } from "../../../store/constants";
+
+const toAPY = (v) => v.toLocaleString(undefined, APY_FORMAT);
+
+const computeRewardAPY = (rewards, decimals, price) =>
+  new Decimal(rewards.reward_per_day)
+    .div(new Decimal(10).pow(decimals))
+    .mul(365)
+    .mul(price)
+    .div(new Decimal(rewards.remaining_rewards).div(new Decimal(10).pow(decimals)))
```

Comment:
I think we should divide not on `remaining_rewards`, but on the (supplied + collateral) or borrowed amount depending on the page.